python/translit/engine.py

```diff
@@ -1,14 +1,16 @@
 from translit.cases import Case
-from translit.converters.translit_cyr_taras_to_lat import convert as convert_cyr_taras_to_lat
+from translit.converters import translit_cyr_taras_to_lat, translit_cyr_nar_to_lat
+
+from collections.abc import Callable, Mapping
```

Comment:
python 3.3+
https://docs.python.org/3/library/collections.abc.html

Reply:
We just need to decide about supported python versions

Reply:
Undid typing.